app/views/teams/new.html.erb

```diff
@@ -1,7 +1,8 @@
 
 <%= render 'components/progress' %>
 <div class="container-fluid">
-<div class="two-column-content">
+  <% unless @team_members.first.nil? %>
```

Comment:
So we said not to merge this one since it would stay the same and the questions will change afterwards correct? just so that we dont have to do double the work fixing it afterwards..